packages/ui/src/utils/getErrorMessageReservedFunds.tsx

```diff
@@ -0,0 +1,30 @@
+export const wikiLinkReservedFunds =
+  'https://github.com/ChainSafe/Multix/wiki/Why-are-funds-reserved%3F'
+
+export const getErrorMessageReservedFunds = (
+  identifier: string,
+  requiredBalanceString?: string,
+  reservedString?: string
+) => {
+  if (!requiredBalanceString) return ''
+
+  const reservedMessage = reservedString ? (
+    <span>
+      Note that it includes ${reservedString} that will be reserved.{' '}
+      <a
+        href={wikiLinkReservedFunds}
+        about="_blank"
+      >
+        More info.
+      </a>
+    </span>
+  ) : (
+    ''
+  )
+  return (
+    <span>
+      The {identifier} doesn't have the required {requiredBalanceString} to submit this transaction.{' '}
+      {reservedMessage}
```

Comment:
Also, it's a small optimisation to render component here ReservedMessage, otherwise it will recalculate reservedMessage every render